frontend/package.json

```diff
@@ -22,18 +22,14 @@
     "@types/testing-library__jest-dom": "^5.14.5",
     "@uswds/uswds": "^3.8.0",
     "apollo-upload-client": "^17.0.0",
-    "browserslist": "4.21.10",
     "classnames": "^2.2.6",
     "flagged": "^2.0.10",
     "google-libphonenumber": "^3.2.33",
     "graphql": "^16.6.0",
     "i18next": "^23.5.1",
     "jwt-decode": "^3.1.2",
-    "loader-utils": "^3.2.1",
     "lodash": "^4.17.21",
-    "minimist": "1.2.8",
     "moment": "^2.29.4",
-    "prop-types": "^15.7.2",
```

Comment:
I believe we need to revert this change

Reply:
@DanielSass reverted and should be good to go 🫡